src/Domain/Server/ServerBase.php

```diff
@@ -66,6 +66,8 @@ public function start(): void {
       ->disableOutput()
 
       ->start();
+
+    return $this->process->getCommandLine();
```

Comment:
Hmm. I don't like returning output here. Given how many data we _could_ return, it seems kind of arbitrary to pick just one. What if we returned the whole process object so client code could select anything it wanted? (Don't forget to update the docblock on the interface accordingly.)